Sources/Subprocess/Platforms/Subprocess+Windows.swift

```diff
@@ -629,6 +750,148 @@ extension Executable {
+            var possiblePaths: Set<String> = []
+            let currentEnvironmentValues = Environment.currentEnvironmentValues()
+            // If `name` does not include extensions, we need to try these extensions
+            var pathExtensions: Set<String> = Set(["com", "exe", "cmd", "bat"])
+            if let extensionList = currentEnvironmentValues["PATHEXT"] {
+                for var ext in extensionList.split(separator: ";") {
+                    ext.removeFirst(1)
+                    pathExtensions.insert(String(ext).lowercased())
+                }
+            }
+            // 1. The directory from which the application loaded.
+            let applicationDirectory = try? fillNullTerminatedWideStringBuffer(
+                initialSize: DWORD(MAX_PATH), maxSize: DWORD(MAX_PATH)
+            ) {
+                return GetModuleFileNameW(nil, $0.baseAddress, DWORD($0.count))
+            }
+            if let applicationDirectory {
+                insertExecutableAddingExtension(
+                    name,
+                    currentPath: applicationDirectory,
+                    pathExtensions: pathExtensions,
+                    storage: &possiblePaths
+                )
+            }
+            // 2. Current directory
+            let directorySize = GetCurrentDirectoryW(0, nil)
+            let currentDirectory = try? fillNullTerminatedWideStringBuffer(
+                initialSize: directorySize >= 0 ? directorySize : DWORD(MAX_PATH),
+                maxSize: DWORD(MAX_PATH)
+            ) {
+                return GetCurrentDirectoryW(DWORD($0.count), $0.baseAddress)
+            }
+            if let currentDirectory {
+                insertExecutableAddingExtension(
+                    name,
+                    currentPath: currentDirectory,
+                    pathExtensions: pathExtensions,
+                    storage: &possiblePaths
+                )
+            }
+            // 3. System directory (System32)
+            let systemDirectorySize = GetSystemDirectoryW(nil, 0)
+            let systemDirectory = try? fillNullTerminatedWideStringBuffer(
+                initialSize: systemDirectorySize >= 0 ? systemDirectorySize : DWORD(MAX_PATH),
+                maxSize: DWORD(MAX_PATH)
+            ) {
+                return GetSystemDirectoryW($0.baseAddress, DWORD($0.count))
+            }
+            if let systemDirectory {
+                insertExecutableAddingExtension(
+                    name,
+                    currentPath: systemDirectory,
+                    pathExtensions: pathExtensions,
+                    storage: &possiblePaths
+                )
+            }
+            // 4. 16 bit Systen Directory
+            // Windows documentation stats that
+            // "No such standard function (similar to GetSystemDirectory)
+            // exists for the 16-bit system folder". Use C:\Windows\System instead
+            let systemDirectory16 = FilePath(#"C:\Windows\System"#)
```

Comment:
There's no guarantee Windows is installed on the C:\ drive.

I think to get the 16-bit Windows directory you need to call GetWindowsDirectoryW (which will give you C:\Windows in most cases), then append a "System" component.